src/main/java/filemanager/directorytracker/ScheduleFileDirectory.java

```diff
@@ -74,6 +79,12 @@ public void goThroughToCheckFile(Job job) {
                     } catch (IOException e) {
                         e.printStackTrace();
                     }
+                    interactionList.forEach((interaction -> {
+                        ExportStats stats = new ExportStats();
+                        stats.setClient(clientName);
+                        stats.setJobId(job.getId());
+                        statsExportService.updateStatsRecords(stats);
+                    }));
```

Comment:
That implementation is pretty good, but I think that we can overload our database with updates in current approach. If you already have a list of interactions, you may just update stats with this list's size.